hack/jenkins/e2e.sh

```diff
@@ -217,6 +217,9 @@ if [[ "${E2E_UP,,}" == "true" ]]; then
         # other.
         export KUBE_SKIP_UPDATE=y
         sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update preview -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update alpha -q" || true
```

Comment:
This PR is fine. Discussed offline with @jlowdermilk - it's a common location for a freshly installed gcloud, which is getting installed often enough that we end up reinstalling components here.

My thinking is actually correct for the normal gcloud on normal runs, though.